bambi/tests/test_built_models.py

```diff
@@ -207,36 +174,21 @@ def test_many_common_many_group_specific(crossed_data):
 
     # build model using fit
     model0 = Model(crossed_data_missing, dropna=True)
-    fitted = model0.fit(
-        "Y ~ continuous + dummy + threecats",
-        group_specific=[
-            "0+threecats|subj",
-            "1|item",
-            "0+continuous|item",
-            "dummy|item",
-            "threecats|site",
-        ],
-        backend="pymc3",
+    model0.fit(
+        "Y ~ continuous + dummy + threecats + (threecats|subj) + (1|item) + (0+continuous|item) + (dummy|item) + (threecats|site)",
         init=None,
         tune=10,
         draws=10,
         chains=2,
     )
-    # model0.build(backend='pymc3')
-    # model0.fit(tune=0, draws=1)
 
-    # build model using add(append=True)
     model1 = Model(crossed_data_missing, dropna=True)
-    model1.add("Y ~ 1")
-    model1.add("continuous")
-    model1.add("dummy")
-    model1.add("threecats")
-    model1.add(group_specific="0+threecats|subj")
-    model1.add(group_specific="1|item")
-    model1.add(group_specific="0+continuous|item")
-    model1.add(group_specific="dummy|item")
-    model1.add(group_specific="threecats|site")
-    model1.build(backend="pymc3")
+    model1.fit(
+        "Y ~ continuous + dummy + threecats + (threecats|subj) + (continuous|item) + (dummy|item) + (threecats|site)",
```

Comment:
The syntax is not the same as with model0, is this intended?

Reply:
Yes. I wanted to see that implicit and explicit group specific intercepts work the same way.